content/_es/policy/about-us.md

```diff
@@ -46,6 +46,6 @@ Durante la administración Obama, se encargó a las agencias gubernamentales que
 
 Login.gov es el resultado de ese llamado a la acción.
 
-El equipo de login.gov se asoció con el [Servicio Digital de los Estados Unidos](https://www.usds.gov/) y [18F](https://18f.gsa.gov/) para aprovechar el exitoso código que ya existía en el sitio web [MyUSCIS](https://my.uscis.gov/), un sitio web que creaba una experiencia segura para los usuarios que necesitaban servicios de tarjeta verde. El equipo también investigó y aplicó lecciones de clientes y empleados de otros proyectos, como connect.gov, que no lograron satisfacer las necesidades de los usuarios. Después de mucha investigación y colaboración, en 2017 se lanzó login.gov como un servicio compartido de confianza para las agencias gubernamentales que se regían por los más altos estándares de seguridad, privacidad y experiencia del usuario. 
+El equipo de login.gov se asoció con el [Servicio Digital de los Estados Unidos(USDS)](https://www.usds.gov/) y [18F](https://18f.gsa.gov/) para aprovechar el exitoso código que ya existía en el sitio web [MyUSCIS](https://my.uscis.gov/), un sitio web que creaba una experiencia segura para los usuarios que necesitaban servicios de tarjeta verde. El equipo también investigó y aplicó lecciones de clientes y empleados de otros proyectos, como connect.gov, que no lograron satisfacer las necesidades de los usuarios. Después de mucha investigación y colaboración, en 2017 se lanzó login.gov como un servicio compartido de confianza para las agencias gubernamentales que se regían por los más altos estándares de seguridad, privacidad y experiencia del usuario. 
```

Comment:
Should there be a space before the parentheses, like in the other languages?

```suggestion
El equipo de login.gov se asoció con el [Servicio Digital de los Estados Unidos (USDS)](https://www.usds.gov/) y [18F](https://18f.gsa.gov/) para aprovechar el exitoso código que ya existía en el sitio web [MyUSCIS](https://my.uscis.gov/), un sitio web que creaba una experiencia segura para los usuarios que necesitaban servicios de tarjeta verde. El equipo también investigó y aplicó lecciones de clientes y empleados de otros proyectos, como connect.gov, que no lograron satisfacer las necesidades de los usuarios. Después de mucha investigación y colaboración, en 2017 se lanzó login.gov como un servicio compartido de confianza para las agencias gubernamentales que se regían por los más altos estándares de seguridad, privacidad y experiencia del usuario. 
```

Reply:
@aduth yes there should be